src/components/dagshub/data-engine/singleFileViewModal/TopButtonsSection.tsx

```diff
@@ -30,12 +37,28 @@ export default function TopButtonsSection({
           width: '100%',
           height: height,
           flexDirection: 'row',
+          gap: '5px',
           alignItems: 'center',
           justifyContent: 'space-between',
           flexShrink: 0,
           flexWrap: isSmallScreen ? 'wrap' : 'nowrap'
         }}
       >
+        {
+            (onSelectItem !== undefined && selected !== undefined) &&
+            <span>
+                <Tooltip
+                    content={"Disable “Select all” to choose specific items"}
```

Comment:
Please make sure Anna agrees with this text. Im not sure if it clear enough that you have to close the modal and disable the "select all" via the gallery itself.

Reply:
I'll ask her

Reply:
She said:
> I think it's minor and not sure how to solve it elegantly...